### PR TITLE
Removes obscure changeling vore

### LINKED ISSUE #19567: (Datums Changeling) Horror Form devour doesn't give you a spell/verb to expel devoured mobs.
If you devour a mob (grab intent>click yourself), there is no spell or verb to regurgitate said mob. Xenos get those, but not horror form lings.

### PR DESCRIPTION
**Pull request by jknpj**
Closes #19567

Seriously, this isn't even in the wiki, what in the good lord.

### CODE REVIEW

**Comment by Irradiation** [comment]
FEEEEATUUUUUUURE

**Comment by Exxion** [comment]
>we can ... immune to stuns

File: code/game/gamemodes/changeling/modularchangling.dm [hunk 1]
```diff
@@ -65,7 +65,7 @@ var/list/datum/power/changeling/powerinstances = list()
 
 /datum/power/changeling/horror_form
 	name = "Horror Form"
-	desc = "This costly evolution allows us to transform into an all-consuming abomination. We are extremely strong, to the point that we can force airlocks open and devour humans whole, and immune to stuns."
+	desc = "This costly evolution allows us to transform into an all-consuming abomination. We are extremely strong, to the point that we can force airlocks, and immune to stuns."
```
Comment (MadmanMartian) [inline code/game/gamemodes/changeling/modularchangling.dm]: "We are incredibly strong, to the point that we can force open airlocks, and are immune to conventional stuns."

**Comment by strawberrytan** [comment]
Wait changelings can eat people whole? Why remove that? Horror mode is shit hitting the fan

**Comment by soyconsumer** [comment]
I don't really think a terrifying monster eating people is vore

**Comment by SGuivenne** [comment]
@soyconsumer it is vore in the same way Timothy Treadwell getting eaten fucking alive by a Grizzly Bear is vore.  It might have vaguely been fetish material, but good luck getting off to it.

**Comment by jknpj** [comment]
TODAY I WILL REMIND THEM

https://github.com/vgstation-coders/vgstation13/blob/d9e848094b6925abf2f76e05e25d483a69edf908/code/modules/mob/living/carbon/human/life/handle_stomach.dm#L3-L38

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
Wait why was this merged
It wasn't even buggy or anything

**Comment by Exxion** [comment]
Look at the branch

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
Still, it's remove it now vs remove it when datums are merged

**Pull request merged**